hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -84,10 +84,6 @@
   uvm_test_seq: chip_sw_base_vseq
   sw_build_device: sim_dv
 
-  // Add default build_opts.
```

Comment:
Is this define no longer needed? Seems like this might break the chip DV sims. The following should also work, if this needs to be kept. 

In this file, do this instead:
```
  prim_default_impl: "prim_pkg::ImplGeneric"
  build_opts: [// Use generic implementations of prim modules
               "+define+PRIM_DEFAULT_IMPL={prim_default_impl}"]
```

For the closed source DV sim, another HJson file will include the above HJSon file and override the `prim_default_impl` value.
```
  overrides: [
    {
      name: prim_default_impl
      value: prim_pkg::Tsmc40ulpef
    }
  ]
```

Reply:
Yes for some reason this still appears to be needed (although I don't fully understand why since the prims all default to the generic implementation).

Your solution sounds good to me.
I'll use that approach and propagate it into foundry.

Reply:
Actually TBH this should not be needed (it should already default to it). Perhaps its better to chase the issue down.

Reply:
Yeah that's what I was initially thinking... let me dig a bit deeper.